src/test/components/MenuButtons.test.js

```diff
@@ -132,7 +132,7 @@ describe('app/MenuButtons', function() {
       expect(getPanel()).toMatchSnapshot();
     });
 
-    it('matches the snapshot for the opened panel for a release profile', () => {
+    fit('matches the snapshot for the opened panel for a release profile', () => {
```

Comment:
Why we changed this?

Reply:
Wooops, that's a mistake. `fit` only runs that one test. I usually do it when debugging a test.